lighthouse-core/test/audits/maskable-icon-test.js

```diff
@@ -5,12 +5,13 @@
  */
 'use strict';
 
-const MaskableIconAudit = require('../../audits/maskable-icon.js');
-const manifestParser = require('../../lib/manifest-parser.js');
+import MaskableIconAudit from '../../audits/maskable-icon.js';
+import manifestParser from '../../lib/manifest-parser.js';
+import {readJson} from '../../../root.js';
 
-const manifestSrc = JSON.stringify(require('../fixtures/manifest.json'));
+const manifestSrc = JSON.stringify(readJson('lighthouse-core/test/fixtures/manifest.json'));
 const manifestWithoutMaskableSrc =
-  JSON.stringify(require('../fixtures/manifest-no-maskable-icon.json'));
+  JSON.stringify(readJson('lighthouse-core/test/fixtures/manifest-no-maskable-icon.json'));
```

Comment:
I'd rather compose `JSON.stringify(readJson(` a few times in tests rather than introduce a new function in root.js, whose existence feels a bit shameful :p

Reply:
should these change to the bare json imports too?

```js
import manifest from 'lighthouse-core/test/fixtures/manifest.json';
import manifestWithoutMaskable = 'lighthouse-core/test/fixtures/manifest-no-maskable-icon.json';

const manifestSrc = JSON.stringify(manifest);
const manifestWithoutMaskableSrc = JSON.stringify(manifestWithoutMaskable);
```